mintpy/prep_fringe.py

```diff
@@ -2,245 +2,312 @@
 ############################################################
 # Program is part of MintPy                                #
 # Copyright (c) 2013, Zhang Yunjun, Heresh Fattahi         #
-# Author: Heresh Fattahi, Forrest Williams, Apr 2020       #
+# Author: Zhang Yunjun, Forrest Williams, Apr 2020         #
 ############################################################
 
 
 import os
+import sys
 import glob
-import h5py
 import argparse
+import h5py
 import numpy as np
-from mintpy.objects import timeseries
-from mintpy.utils import ptime, readfile, writefile, isce_utils, utils
+import xml.etree.ElementTree as ET
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Using xml.etree.ElementTree to parse untrusted XML data is known to be vulnerable to XML attacks. Replace xml.etree.ElementTree with the equivalent defusedxml package, or make sure defusedxml.defuse_stdlib() is called.](https://app.codacy.com/manual/insarlab/MintPy/pullRequest?prid=5496251)